MobSF/utils.py

```diff
@@ -630,4 +635,37 @@ def is_zip(self):
     def is_ipa(self):
         return (self.file_type in settings.IPA_MIME) and self.file_name_lower.endswith('.ipa')
     def is_appx(self):
-        return (self.file_type in settings.APPX_MIME) and self.file_name_lower.endswith('.appx')
\ No newline at end of file
+        return (self.file_type in settings.APPX_MIME) and self.file_name_lower.endswith('.appx')
+
+
+
+def request_method(methods):
+    """
+    :param methods http method
+    need django HttpRequest
+    """
+    def decorator(func):
+        @functools.wraps(func)
+        def wrapper(*args, **kwargs):
+
+            if not isinstance(methods, list) and not isinstance(methods, tuple):
+                raise ValueError('the parameter methods is not a list or tuple')
+            
+            methods_upper = [m.upper() for m in methods]
+            for method in methods_upper:
+                if method not in ALLOW_METHODS:
+                    raise ValueError('the methods is not allow')

```

Comment:
Please change it to `This method is not allowed`

Reply:
ok